mantidproject/mantid
# Adding solid cylinder integral tests

~~**Do not merge before #29579**~~

**Description of work.**

Uses the function added in #29579 to make unit tests for evaluating line integrals at different points in a solid cylinder, along both the y and z axis.



**To test:**



Code review and making sure the unit tests provide good coverage for a solid cylinder. Verify that GeometryTest can be built and run to make sure all tests pass.

See #29579.

*This does not require release notes* because **it is an internal change.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
